integration-tests/todo_write.test.ts

```
@@ -0,0 +1,132 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { describe, it, expect } from 'vitest';
+import { TestRig, printDebugInfo, validateModelOutput } from './test-helper.js';
+
+describe('todo_write', () => {
+  it('should be able to create and manage a todo list', async () => {
+    const rig = new TestRig();
+    await rig.setup('should be able to create and manage a todo list');
+
+    const prompt = `I want to implement a new feature to track user preferences. Here are the tasks:
```

Could we add a prompt example that doesn't specify a todo task explicitly?

---

This is integration test which use real api endpoint for testing. We can keep it simple so that it won't fail unexpectedly.